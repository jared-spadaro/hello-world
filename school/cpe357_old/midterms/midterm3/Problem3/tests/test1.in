newList
checkSpace 0 
addAtIndex 0 1.11
printList 1
checkSpace 1
done
