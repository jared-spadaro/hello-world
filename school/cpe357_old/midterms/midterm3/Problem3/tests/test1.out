checkSpace: Expected amount of memory allocated
List: 1.11 
checkSpace: Expected amount of memory allocated
